paasta_tools/tron/tron_timeutils.py

```diff
@@ -28,6 +33,21 @@ def delta_total_seconds(td):
     return (microseconds + (seconds + days * 24 * 3600) * 10**6) / 10**6
 
 
+def check_timedelta(value: Any, param_name: str) -> Tuple[bool, str]:
```

Comment:
This is stricter than Tron's validation. This only allows d, h, m, s (for days, hours, minutes, seconds), while Tron allows any of ['m', 'min', 'mins', 'minute', 'minutes'] for minutes, and similar things for the other units. 

I was trying to avoid copying too much complex validation code from Tron. Was thinking we could do a similar thing (i.e. being less flexible than Tron) for fields like `schedule`.

Reply:
This sounds good to me. I think tron's `schedule` field is unnecessarily flexible, but in a different way.

Are you planning on preemptively fixing the MASTER files that we have already?

Reply:
Yeah, I would fix anything that doesn't pass validation when I changed the pre-commit hook.